src/types/module-deployments/NewSplitRecipient.ts

```diff
@@ -0,0 +1,26 @@
+import { BigNumber } from "ethers";
+
+/**
+ * The NewSplitRecipient interface describes the structure of a new split recipient.
+ * The `shares` property is important for the calculation of the the total split.
+ *
+ * If there are two recipients each with 10 shares, they each will receive 50%
+ * of the total royalties.
+ *
+ * If there are two recipients each with 1 share, they each will receive 50%
+ * of the total royalties.
+ *
+ * I.e. the total number of shares is used to calculate the percentage of the
+ * total royalties that is allocated to each recipient.
+ */
+export interface NewSplitRecipient {
```

Comment:
does the shares mechanism made sense for developers? if you're a new dev using this. would you understand what's the shares and how to split them based of total number of shares? (just wondering)

Reply:
does it make sense to be `NewSplitRecipient` or `SplitRecipient` is fine

Reply:
I would assume so. Its not all that complicated and if we have good docs/tutorials, it should click very quickly. Otherwise the other option is using percentages which is its own mess that is not as clean imo.

Reply:
wouldn't this just be `SplitRecipient`? the "new" is obvious if you're *adding* it, no? does the `get` on this return an `OldSplitRecipient` interface?

Reply:
Its different. The `NewSplitRecipient` takes in an integer shares value, and the `SplitRecipient` returns back the % value we could make them both percentages, but then they have to ensure what they create always adds up to 100%